docs/test_artifacts.md

```diff
@@ -74,3 +74,69 @@ To export path to your local flank repository just source [.env](../.env) file.
 > As a developer I want to remove test artifacts
 
 * Run `flankScripts testArtifacts remove` this will remove the remote copy of test artifacts for current working branch.
+
+
+## iOS test artifacts
+
+Currently we have 4 different iOS test projects:
+
+1. EarlGreyExample
+2. FlankExample
+3. FlankGameLoopExample
+4. FlankTestPlansExample
+
+Source code of each of them is located under: [test_projects/ios](../test_projects/ios)
+
+Test artifacts for each project contains: 
+* build output in Debug-iphoneos
+* zipped build output in PROJECT_NAME.zip,
+* .xctestrun file for each test target
+
+### EarlGreyExample
+
+This project is basically clone of [EarlGrey](https://github.com/google/EarlGrey). 
+Source project contains two test targets: EarlGreyExampleSwiftTests, EarlGreyExampleTests.
```

Comment:
Additionally do not forget to add an explanation that this build may fail on some configurations,
create an issue about a problem and past a link here in the documentation.

Reply:
@jan-gogo Figured out solution - build shouldn't fail on other devices.
EarlGreyExample requires Xcode's "legacy build system", I added correct parameter in flank-scripts to force that, since it couldn't be specified in project itself.